R/util-MakeSnapshotHelpers.R

```diff
@@ -246,35 +248,36 @@ MakeRptStudyDetails <- function(lResults, status_study, gsm_analysis_date) {
   }
   status_study %>%
     mutate(
-      snapshot_date = gsm_analysis_date,
-      num_of_sites_flagged = num_of_sites_flagged,
-      enrolling_sites_with_flagged_kris = as.integer(0),
-      study_age = ExtractStudyAge(.data$fpfv, .data$snapshot_date),
-      pt_cycle_id = NA_character_,
-      pt_data_dt = NA_character_
+      "snapshot_date" = gsm_analysis_date,
+      "num_of_sites_flagged" = num_of_sites_flagged,
+      "enrolling_sites_with_flagged_kris" = as.integer(0),
+      "study_age" = ExtractStudyAge(.data$fpfv, .data$snapshot_date),
+      "pt_cycle_id" = NA_character_,
+      "pt_data_dt" = NA_character_
     ) %>%
-    select(
-      "study_id" = "studyid",
-      "snapshot_date",
-      "protocol_title" = "title",
-      "therapeutic_area" = "ta",
-      "indication",
-      "phase",
-      "product",
-      "enrolled_sites",
-      "enrolled_participants",
-      "planned_sites",
-      "planned_participants",
-      "study_status" = "status",
-      "fpfv",
-      "lpfv",
-      "lplv",
-      "study_age",
-      "num_of_sites_flagged",
-      "enrolling_sites_with_flagged_kris",
-      "pt_cycle_id",
-      "pt_data_dt"
-    )
+    select("studyid",
+           "snapshot_date",
+           "title",
+           "ta",
+           "indication",
+           "phase",
+           "product",
+           "enrolled_sites",
+           "enrolled_participants",
+           "planned_sites",
+           "planned_participants",
+           "est_fpfv",
+           "est_lpfv",
+           "est_lplv",
+           "status",
+           "fpfv",
+           "lpfv",
+           "lplv",
+           "study_age",
+           "num_of_sites_flagged",
+           "enrolling_sites_with_flagged_kris",
+           "pt_cycle_id",
+           "pt_data_dt")
 }
```

Comment:
It might be worth looking at a helper function to select columns in the order defined by `gsm::rbm_data_spec` but that might be more confusing than having a verbose `select()` statement.